### DISCUSSION TITLE
Vcard still shows WORK and HOME labels

### DISCUSSION THREAD

**Discussion by allesoverwaterstof** [JulianPrieber/littlelink-custom#831] [Feature requests]
### LinkStack version

4.7.6

### Description

Vcards are still not properly generated. The plugin seems not tpo be working properly and the Github plugin author does not react to comments anymoren

### Details about your system

php 8

### How to reproduce

just create an vcard and then try to import it on your phone or macbook

### Possible Solution

Please use another Vcard plugin

**Comment by JulianPrieber** [comment]
Known issue.
Unless there is a better alternative out there, there is not much that I can do right now.

**Comment by ciroiriarte** [comment]
could you explain what's the problem?, it's a Mac only thing?

**Comment by allesoverwaterstof** [comment]
No, its a common problem that is caused by the VCard module in your solution



Test it for yourself and you will see






---- Op Wed, 01 May 2024 06:31:08 +0200 schreef Ciro Iriarte ***@***.***> ----





could you explain what's the problem?, it's a Mac only thing?

—
Reply to this email directly, https://github.com/LinkStackOrg/LinkStack/issues/726#issuecomment-2087960767, or https://github.com/notifications/unsubscribe-auth/AUYUX5BKD7XMABNUPIOXBVDZABVYZAVCNFSM6AAAAABDIB7CWOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOBXHE3DANZWG4.
You are receiving this because you authored the thread.









https://allesoverwaterstof.nl 







Frank Mietes

 

***@***.*** 

+31 6 1616 6364 

http://www.allesoverwaterstof.nl 






https://twitter.com/alleswaterstof 



https://www.linkedin.com/company/allesoverwaterstof/ 



https://www.instagram.com/allesoverwaterstof/ 



https://www.facebook.com/allesoverwaterstof/ 





https://www.youtube.com/channel/UCh7sCwM65lJo5u1W2KaLkJw 










https://allesoverwaterstof.nl/nieuwsbrief/










_____________________________



De informatie in deze mail is uitsluitend bestemd voor de geadresseerde en bevat mogelijk vertrouwelijke informatie. Het is voor een ander dan de geadresseerde niet toegestaan de informatie openbaar te maken, te kopiëren, te verspreiden of anderszins te gebruiken. Indien deze mail u bij vergissing bereikt, stel dan de afzender hiervan op de hoogte en verwijder de mail van uw systeem. De inhoud van de e-mail kan onjuist of onvolledig zijn. Aan de inhoud van de e mail kunnen geen rechten ontleend worden.

The information in this e-mail is meant for the exclusive use of the addressee and may contain confidential information. No part of this information may be made public, copied, spread, or used in other way, by anyone apart from the addressee. If this mail should reach you by accident, inform the sender and remove the mail from your system. The contents of the e-mail may be wrong or incomplete. No rights can be derived from the contents of the e-mail.

**Comment by dridhas** [comment]
Hello, same issue still persists on 4.8.0
Any chance to hide one if its empty?

**Comment by JulianPrieber** [comment]
Not that I know of. If all things go well, we'll be able to address this issue in version 5.0.
